woocommerce/woocommerce-gutenberg-products-block
# Store Customization > Add the Featured Category Cover Image pattern

## What

The **Featured Category Cover Image** pattern was originally removed on https://github.com/woocommerce/woocommerce-blocks/pull/10239, but we need to reintroduce it as it is a requirement to compose the homepage templates for the Pattern Assembler as designed on Mrk6SERPZ4KrFHSjM0a8TK-fi-1406_127404 and we can't rely exclusively on blocks for having AI-generated content.

This is a wireframed pattern: in other words, it doesn't have any opinionated styles regarding colors for fonts or backgrounds.

Fixes #11107

## Why



This pattern is required to compose two different templates within the Patterns Assembler.

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Create a new post
2. Insert the **Featured Category Cover Image** pattern
3. Make sure the styles match what is on the screenshots of this PR
4. Save the post and check the front end: make sure things are working as expected as well


## Screenshots or screencast



| Desktop | Mobile |
| ------ | ----- |
| <img width="1214" alt="Screenshot 2023-10-03 at 15 39 56" src="https://github.com/woocommerce/woocommerce-blocks/assets/15730971/cc1a509b-0e4f-448a-8483-0844d9d0068d"> | <img width="377" alt="Screenshot 2023-10-03 at 15 39 45" src="https://github.com/woocommerce/woocommerce-blocks/assets/15730971/68317e43-211c-464e-a511-7a3368a5cbab"> |

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Add the Featured Category Cover Image pattern
